cfme/tests/containers/test_containers_summary.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import StatusBox
+from utils import testgen
+from utils.version import current_version
+import time
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('has_no_container_providers'),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+
+# container_dashboard
+# CMP-9827 # CMP-9826 # CMP-9825 # CMP-9824 # CMP-9823 # CMP-9822 # CMP-9821 # CMP-9820
+SUMMARY_OBJECTS = \
+    ['routes', 'projects', 'images', 'registries', 'services', 'containers', 'pods', 'nodes']
+
+
+def test_containers_summary_objects(provider):
+    """ Containers overview page > Widgets > Widgets summary
+       This test checks that the amount of a selected object in the system is shown correctly
+        in the widgets in the
+       Overview menu
+       Steps:
+           * Goes to Compute --> Containers --> Overview
+           * Checks how many objects are shown in the selected widget
+           * Goes to Containers summary page and checks how many objects are shown there.
+           * Checks the amount is equal
+       """
+    sel.force_navigate('container_dashboard')
+    sel.force_navigate('containers_provider', context={'provider': provider})
+    routes_val = provider.summary.relationships.routes.value
+    projects_val = provider.summary.relationships.projects.value
+    images_val = provider.summary.relationships.images.value
+    image_registries_val = provider.summary.relationships.image_registries.value
+    services_val = provider.summary.relationships.services.value
+    containers_val = provider.summary.relationships.containers.value
+    pods_val = provider.summary.relationships.pods.value
+    nodes_val = provider.summary.relationships.nodes.value
+    for summary_object in SUMMARY_OBJECTS:
+        sel.force_navigate('container_dashboard')
+        time.sleep(2)
```

Comment:
`time.sleep()` is also discouraged. It may work for you but if the test is ran on a machine that is on a different network, 2 seconds might not be enough and you will see cryptic errors...